app/assets/src/components/common/DetailsSidebar/SampleDetailsMode/MetadataTab.jsx

```diff
@@ -119,7 +119,10 @@ class MetadataTab extends React.Component {
     return val === undefined || val === null || val === "" ? (
       <div className={cs.emptyValue}>--</div>
     ) : (
-      <div className={cs.metadataValue}>{val}</div>
+      <div className={cs.metadataValue}>
+        {/* If asked to display an arbitrary object, try the name field */}
+        {isObject(val) && val.name ? val.name : val}
```

Comment:
Does it still make sense to display the object? In what circumstances would that happen?

Reply:
probably better not to try to display the object (you'll get `Objects are not valid as a React child`). I'll change to `isObject(val) ? val.name : val`